src/content/fr/2019/seo.md

```diff
@@ -0,0 +1,421 @@
+Au cours de nos recherches, nous avons constaté que seuls 14,67% des sites sont éligibles pour des résultats riches sur mobile. Fait intéressant, l'admissibilité au site de bureau est légèrement inférieure à 12,46%. Cela suggère que les propriétaires de sites peuvent faire beaucoup plus pour optimiser la façon dont leurs pages d'accueil apparaissent dans la recherche.
+Parmi les sites avec un balisage de données structuré, les cinq types les plus implémentés sont:
+1. «webSite» (16,02%)
+2. «SearchAction» (14,35%)
+3. «Organisation» (12,89%)
+4. `webPage` (11,58%)
+5. «ImageObject» (5,35%)
+Fait intéressant, l'un des types de données les plus populaires qui déclenche une fonctionnalité de moteur de recherche est «SearchAction», qui alimente la [boîte de recherche des liens annexes](https://developers.google.com/search/docs/data-types/sitelinks-searchbox) .
+Les cinq principaux types de balisage conduisent tous à une plus grande visibilité dans les résultats de recherche de Google, ce qui pourrait être le facteur d'une adoption plus répandue de ces types de données structurées.
+Étant donné que nous n'avons examiné que les pages d'accueil, les résultats pourraient sembler très différents si nous considérions également les pages intérieures.
+Les étoiles d'avis ne se trouvent que sur 1,09% des pages d'accueil du web (via [AggregateRating](https://schema.org/AggregateRating)). En outre, le [QAPage] nouvellement introduit (https://schema.org/QAPage) n'est apparu que dans 48 cas, et le [FAQPage](https://schema.org/FAQPage) à une fréquence légèrement plus élevée de 218 fois. Ces deux derniers décomptes devraient augmenter à l'avenir alors que nous effectuons davantage d'analyses et approfondissons l'analyse Web Almanac.
+
+### Internationalisation
+
+L'internationalisation est l'un des aspects les plus complexes du référencement naturel, même [selon certains employés de la recherche Google](https://twitter.com/JohnMu/status/965507331369984002). L'internationalisation du référencement se concentre sur la diffusion du bon contenu à partir d'un site web avec plusieurs versions linguistiques ou nationales et sur le ciblage du contenu vers la langue et l'emplacement spécifiques de l'utilisateur.
+Alors que 38,40% des sites de bureau (33,79% sur mobile) ont l'attribut HTML lang réglé sur anglais, seulement 7,43% (6,79% sur mobile) des sites contiennent également un lien «hreflang» vers une autre version linguistique. Cela suggère que la grande majorité des sites web que nous avons analysés n'offrent pas de versions distinctes de leur page d'accueil qui nécessiteraient un ciblage linguistique - sauf si ces versions distinctes existent mais n'ont pas été configurées correctement
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th><code>hreflang</code></th>
```

Comment:
P-e ajouter `scope="col"` aux `th`? #a11y :)

Reply:
I suggest we add this scope="col" and scope="row" in another issue for all the site. You could create an issue if interested :)

Reply:
In #660 we did fix the more complex tables (where this clarification matters most) after you raised #657 @nico3333fr , but we left the simple tables as was so they could stay as markdown instead of having to HTML them all. This one's a bit weird in that it's a simple table, but is HTML already. This looks to be mainly to allow adding a `class="numeric"` attribute to some of the cells (which can't be done with markdown).

So we could add these here but not sure it's worth it to be honest. And we'd need to update English and Japanese too if doing that for consistency.

I suggest, as it's a simple table, we leave as is for now. Raise a separate PR if we want to fix it and see if any other examples across the site like @AymenLoukil suggests.